### PR TITLE
wren lang. programming (new formula)

### PR DESCRIPTION
**Pull request by alexandregz**

### CODE REVIEW

File: wren.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Wren < Formula
+  homepage "http://munificent.github.io/wren/"
+  url "https://github.com/munificent/wren/archive/efd161cea278e5cc9d6b404451443cf48d4057a8.tar.gz"
```
Comment (DomT4) [inline wren.rb]: This should be ` head "https://github.com/munificent/wren.git" `

File: wren.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  sha1 "f8757ad1b3dbe312c2928e00e095007d87b97748"
```
Comment (DomT4) [inline wren.rb]: And consequent to the line above, this can be binned.

File: wren.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+  depends_on "pkg-config" => :build
+
+  def install
+    system "make"
+    bin.install "wren"
+    prefix.install Dir["Makefile", "include", "script", "src", "test"]
```
Comment (DomT4) [inline wren.rb]: ` include ` can go in the prefix. Everything else should go in ` libexec `.

File: wren.rb (hunk 1 above)
Comment (alexandregz) [inline wren.rb]: ok

File: wren.rb (hunk 2 above)
Comment (alexandregz) [inline wren.rb]: ok (2) :-)

File: wren.rb (hunk 3 above)
Comment (alexandregz) [inline wren.rb]: you're right (3)

File: wren.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  depends_on "pkg-config" => :build
+
+  def install
+    system "make"
+    bin.install "wren"
+    libexec.install Dir["Makefile", "include", "script", "src", "test"]
+  end
+
+  test do
+    libexec.cd do
```
Comment (mistydemeo) [inline wren.rb]: Minor stylistic nit:

```ruby
cd libexec do
```

Comment (alexandregz) [inline wren.rb]: oka

File: wren.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
+class Wren < Formula
+  homepage "http://munificent.github.io/wren/"
+  head "https://github.com/munificent/wren.git"
+
+  depends_on "pkg-config" => :build
+
+  def install
+    system "make"
+    bin.install "wren"
+    libexec.install Dir["Makefile", "include", "script", "src", "test"]
```
Comment (DomT4) [inline wren.rb]: ` include ` might be useful in the ` prefix ` still.

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
There's some failure here on the test-bot which needs addressing:

```
==> audit problems
homebrew/head-only/wren:
 * Formula should have a desc (Description).
 * Please use https:// for http://munificent.github.io/wren/
 * Dir["include"] is unnecessary; just use "include"
```

```
Passed: [32m590[0m Failed: [31m0[0m Skipped: [33m9[0mTraceback (most recent call last):
  File "./script/test.py", line 284, in <module>
    walk(join(WREN_DIR, 'example'), run_example)
  File "./script/test.py", line 61, in walk
    for file in [file for file in listdir(dir) if not file in ignored]:
OSError: [Errno 2] No such file or directory: '/usr/local/Cellar/wren/HEAD/libexec/example'
make: *** [test] Error 1
Error: homebrew/head-only/wren: failed
Failed executing: make test
```

**Comment by DomT4** [comment]
Closing as dead. Please open a new updated PR if you'd like to.

**Pull request closed without merging**